exporters/otlp/otlplog/otlploghttp/exporter.go

```diff
@@ -47,12 +46,11 @@ func (e *Exporter) Export(ctx context.Context, records []log.Record) error {
 	if e.stopped.Load() {
 		return nil
 	}
-	otlp, err := transformResourceLogs(records)
+	otlp := transformResourceLogs(records)
 	if otlp != nil {
-		// Best effort upload of transformable logs.
-		err = errors.Join(err, e.client.Load().UploadLogs(ctx, otlp))
+		return e.client.Load().UploadLogs(ctx, otlp)
 	}
-	return err
+	return nil
```

Comment:
Can we just refactor this to:

```go
if otlp == nil {
  return nil
}

return e.client.Load().UploadLogs(ctx, otlp)
```

Reply:
I'm not sure how this changes anything, but I can apply it.

Reply:
"usual" code flow is not indented

Reply:
I'm not sure the compiler cares :laughing:

Reply:
It does not. Humans care  😁Similarly to Error Indent Flow